sources/Launcher/DYNRobustnessAnalysisLauncher.cpp

```diff
@@ -97,6 +109,8 @@ RobustnessAnalysisLauncher::init() {
     throw DYNAlgorithmsError(DirectoryDoesNotExist, workingDirectory_);
   workingDirectory_ += '/';  // to be sure to have an '/' at the end of the path
 
+  initLog();
```

Comment:
Maybe we should add a boolean with a local value for each existig algorithms to avoid creating empty logs for SA/CS...

Reply:
Why ?
The primary idea was to log version informations for all three algorithms and not only MC !
Moreover, it initialize the log file, so developers of algorithms can subsequently add logs for SA and CS too.

Reply:
yes but I'm not sure it will not break the patform integration (we had to modify something in the code when we first created the MC log)

Reply:
OK, new parameter to conditionally initialize log.  
For the moment, this is enabled only for Margin Calculation.